<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxExampleClass</class>
 <widget class="QWidget" name="MessageBoxExampleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessageBoxExample</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="infoBtn">
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="warningBtn">
     <property name="text">
      <string>Warning</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="questionBtn">
     <property name="text">
      <string>question</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="criticalBtn">
     <property name="text">
      <string>critical</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="infoBtn_2">
     <property name="text">
      <string>Information 3</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="warningBtn_2">
     <property name="text">
      <string>Warning 5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="criticalBtn_2">
     <property name="text">
      <string>critical 10</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="questionBtn_2">
     <property name="text">
      <string>question 20</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MessageBoxExample.qrc"/>
 </resources>
 <connections/>
</ui>
